src/main/java/edu/harvard/iq/dataverse/datasetutility/AddReplaceFileHelper.java

```diff
@@ -1192,7 +1196,7 @@ private boolean step_030_createNewFilesViaIngest(){
 
         // Load the working version of the Dataset
         workingVersion = dataset.getEditVersion();
-        clone =   workingVersion.cloneDatasetVersion();
+        clone =   workingVersion.cloneDatasetVersion(licenseServiceBean.getDefault());
```

Comment:
Ah - so if clone just gets the working version license this goes away, this class doesn't need licenseServer, the Files API class doesn't need it to pass into here, etc.